src/com/github/igotyou/FactoryMod/FactoryModManager.java

```diff
@@ -29,13 +29,17 @@
 	private HashSet<Material> possibleInteractionBlock;
 	private Material factoryInteractionMaterial;
 	private boolean citadelEnabled;
+	private int redstonePowerOn;
+	private int redstoneRecipeChange;
 	private String compactLore;
 
 	public FactoryModManager(FactoryModPlugin plugin,
-			Material factoryInteractionMaterial, boolean citadelEnabled) {
+			Material factoryInteractionMaterial, boolean citadelEnabled, int redstonePowerOn, int redstoneRecipeChange) {
```

Comment:
It's fine for now aa long as we only have one factory type. If it turns out we dont need it for other factories we can still move it later